### ISSUE TITLE
sql/schema: Failed to Create Schema

### ISSUE THREAD

**Issue by jaredallard** [facebook/ent#818]
**What Happened**: When I run `Create()` twice it fails the second time. I get the following error message:

```
sql/schema: create index \"media\": ERROR: relation \"media_source_source_uri\" already exists (SQLSTATE 42P07)
```

**How To Reproduce**: Here's my [schema definition](https://github.com/tritonmedia/api/blob/master/internal/ent/schema/media.go) and a [repository](https://github.com/tritonmedia/api) where this can be reproduced, if needed.

```
docker-compose up -d
make
./bin/api
```

**Details**:

 * **Driver**: Postgres 12.4 (pgx v4.9.0)
 * **Ent**: v0.4.3 (also happened on v0.4.2)

**Comment by a8m** [comment]
Thanks for reporting this issue @jaredallard. I'll give it a look.

**Comment by a8m** [comment]
Thanks for your engagement @jaredallard. Issue should be solved with #828.
There was an issue in the Postgres migration code that affected only tables with uncountable names, like "media" or "equipment" (no plural form). 

Please let me know if you still have troubles.

**Comment by a8m** [comment]
Thanks for your collaboration @jaredallard. Issue should be solved with #828.
There was an issue in the Postgres migration code that affected only tables with uncountable names, like "media" or "equipment" (no plural form). 

Please let me know if you still have troubles.
